### PR TITLE
BRANCH-41 add more souts

### PR DESCRIPTION
**Pull request by voitsel**
with comment

### CODE REVIEW

File: src/main/java/NewClass.java [hunk 1]
```diff
@@ -2,5 +2,7 @@
     public static void main(String[] args) {
         System.out.println("Test");
         System.out.println("Test");
```
Comment (kara4k) [inline src/main/java/NewClass.java:4]: review comment on line 1

File: src/main/java/NewClass.java [hunk 2]
```diff
@@ -2,5 +2,7 @@
...
+        System.out.println("Test");
+        System.out.println("Test");
```
Comment (kara4k) [inline src/main/java/NewClass.java:6]: review comment on line 2

File: src/main/java/NewClass.java [hunk 3]
```diff
@@ -2,5 +2,7 @@
     public static void main(String[] args) {
         System.out.println("Test");
         System.out.println("Test");
+        System.out.println("Test");
```
Comment (kara4k) [inline src/main/java/NewClass.java:5]: review comment 3

**Review by kara4k — CHANGES_REQUESTED** [review]

**Comment by voitsel** [comment]
default comment

File: src/main/java/NewClass.java (hunk 2 above)
Comment (kara4k) [reply src/main/java/NewClass.java:6]: one more

**Review by kara4k — APPROVED** [review]

File: src/main/java/NewClass.java (hunk 1 above)
Comment (kara4k) [reply src/main/java/NewClass.java:4]: approved

**Review by kara4k — APPROVED** [review]
Nice

**Review by kara4k — COMMENTED** [review]
aaaaa

**Pull request merged**